Volume/Views/MainView/PublicationList.swift

```diff
@@ -162,7 +176,17 @@ extension PublicationList {
     )
     private enum PublicationListState {
         case loading
+        case reloading(Results)
         case results(Results)
+        
+        var shouldDisableScroll: Bool {
```

Comment:
Since you put this here, I would put `isLoading` (line 55) here too. And maybe even `someFollowedPublications` (line 46).